src/platform/site-wide/cta-widget/helpers.js

```diff
@@ -59,6 +61,30 @@ export const mhvBaseUrl = () => {
   return `https://${mhvSubdomain}.myhealth.va.gov`;
 };
 
+const mhvToEauthRoutes = {
+  'download-my-data': 'download_my_data',
+  'web/myhealthevet/refill-prescriptions': 'prescription_refill',
+  'secure-messaging': 'secure_messaging',
+  appointments: 'appointments',
+  'labs-tests': 'lab-tests',
```

Comment:
Just calling out that this URL is unconfirmed as a viable deep link target - we're awaiting confirmation from MHV that they can support this.